themes/cclw/pages/terms-of-use.tsx

```diff
@@ -185,7 +185,9 @@ const TermsOfUse = () => {
             <p>
               The CPR Database and App contains data that were created by third parties. The data includes document summaries that were written by
               third parties and/or information about the documents (”meta-data”) assigned by third parties, e.g. classifications to sector or legal
-              principle.
+              principle, and Law and Policy document curation and summaries; document metadata from{" "}
```

Comment:
might be one for Alan, @PatFawbertMills but not sure if this is how he intended the copy to be added to the terms, it wasn't all too clear in the notion doc.
![Screenshot 2024-08-14 at 17 08 09](https://github.com/user-attachments/assets/05ddee14-870c-437e-b148-f8305ad0854c)

Reply:
I think this is in the wrong place. Judging by Alan's notion page the update needs to be made to the table below where we have the list of sources

Reply:
cool cool, checked with alan+ peter and updated it 
![Screenshot 2024-08-19 at 12 00 28](https://github.com/user-attachments/assets/6468f974-88a4-49b9-bda3-f387d37adb1c)